src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -640,7 +651,11 @@ void LedgerConsensusImp::timerEntry ()
         using namespace std::chrono;
         mCurrentMSeconds = duration_cast<milliseconds>
                            (steady_clock::now() - mConsensusStartTime);
-        mClosePercent = mCurrentMSeconds * 100 / mPreviousMSeconds;
+
+        mClosePercent = mCurrentMSeconds * 100 /
+            std::min (
+                mPreviousMSeconds,
+                duration_cast<milliseconds>(AV_MIN_CONSENSUS_TIME));
```

Comment:
```
    // How long the close has taken, expressed as a percentage of the time that
    // we expected it to take.
    int mClosePercent;
```

It's perfectly reasonable to exceed 100% of the previous close time. In fact, if we predict the close time perfectly, about half the time it will take longer than we expected.